pkg/monitor/format/format.go

```diff
@@ -122,6 +122,16 @@ func (m *MonitorFormatter) traceEvents(prefix string, data []byte) {
 	}
 }
 
+func (m *MonitorFormatter) traceSockEvents(prefix string, data []byte) {
+	tn := monitor.TraceSockNotify{}
+
+	binary.Read(bytes.NewReader(data), byteorder.Native, &tn)
+	// TODO: Currently only printed with the debug option. Extend it to info and json.
```

Comment:
I'm curious, what stops us from extending to info/json? Mainly asking just because usually what happens when someone commits a `TODO` into the codebase is that we promptly forget it and never fix the todo.

(There's a school of thought that you should always mitigate this by writing something like `TODO(aditighag)`, that's marginally better since we know who to ask directly about the TODO, but that doesn't ensure the todo is actually addressed..)

Reply:
As the commit description says, there is no filtering or monitor aggregation support at this time so we want to dump the parser only with debug mode.

Reply:
Linked to the GH issue for follow-ups.